diff --git a/src/AppBundle/Form/GenusFormType.php b/src/AppBundle/Form/GenusFormType.php
new file mode 100644
index 0000000..bf8a4db
--- /dev/null
+++ b/src/AppBundle/Form/GenusFormType.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class GenusFormType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+
+    }
+}
