diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/AppBundle/Controller/Admin/GenusAdminController.php
index d0823d1..5206bd3 100644
--- a/src/AppBundle/Controller/Admin/GenusAdminController.php
+++ b/src/AppBundle/Controller/Admin/GenusAdminController.php
@@ -5,6 +5,7 @@ namespace AppBundle\Controller\Admin;
 use AppBundle\Form\ProductFormType;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
 
 /**
  * @Route("/admin")
@@ -28,10 +29,16 @@ class GenusAdminController extends Controller
     /**
      * @Route("/genus/new", name="admin_genus_new")
      */
-    public function newAction()
+    public function newAction(Request $request)
     {
         $form = $this->createForm(ProductFormType::class);
 
+        // only handles data on POST
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            dump($form->getData());die;
+        }
+
         return $this->render('admin/genus/new.html.twig', [
             'genusForm' => $form->createView()
         ]);
