diff --git a/app/Resources/views/genus/show.html.twig b/app/Resources/views/genus/show.html.twig
index b011382..6d95989 100644
--- a/app/Resources/views/genus/show.html.twig
+++ b/app/Resources/views/genus/show.html.twig
@@ -14,7 +14,7 @@
                 <dt>Known Species:</dt>
                 <dd>289</dd>
                 <dt>Fun Fact:</dt>
-                <dd>Octopuses can change the color of their body in just three-tenths of a second!</dd>
+                <dd>{{ funFact }}</dd>
             </dl>
         </div>
     </div>
diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 8a363b7..5b0bcdb 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -15,8 +15,11 @@ class GenusController extends Controller
      */
     public function showAction($genusName)
     {
+        $funFact = 'Octopuses can change the color of their body in just *three-tenths* of a second!';
+
         return $this->render('genus/show.html.twig', array(
             'name' => $genusName,
+            'funFact' => $funFact,
         ));
     }
 
