diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index 3a375c4..ec262f9 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -46,6 +46,8 @@
             Add Another Scientist
         </a>
     </div>
+    {# prevents weird label from showing up in new #}
+    {{ form_widget(genusForm.genusScientists) }}
 
     <button type="submit" class="btn btn-primary" formnovalidate>Save</button>
 {{ form_end(genusForm) }}
