diff --git a/src/AppBundle/Repository/GenusRepository.php b/src/AppBundle/Repository/GenusRepository.php
index 0a0a3e9..a77abd8 100644
--- a/src/AppBundle/Repository/GenusRepository.php
+++ b/src/AppBundle/Repository/GenusRepository.php
@@ -24,6 +24,17 @@ class GenusRepository extends EntityRepository
             ->execute();
     }
 
+    /**
+     * @return Genus[]
+     */
+    public function findAllExperts()
+    {
+        return $this->createQueryBuilder('genus')
+            ->addCriteria(self::createExpertCriteria())
+            ->getQuery()
+            ->execute();
+    }
+
     static public function createExpertCriteria()
     {
         return Criteria::create()
