diff --git a/src/AppBundle/Form/ProductFormType.php b/src/AppBundle/Form/ProductFormType.php
index cc5c1d7..aece5c2 100644
--- a/src/AppBundle/Form/ProductFormType.php
+++ b/src/AppBundle/Form/ProductFormType.php
@@ -31,7 +31,9 @@ class ProductFormType extends AbstractType
                     'No' => false,
                 ]
             ])
-            ->add('firstDiscoveredAt')
+            ->add('firstDiscoveredAt', null, [
+                'widget' => 'single_text'
+            ])
         ;
     }
 
