diff --git a/src/AppBundle/Form/GenusFormType.php b/src/AppBundle/Form/GenusFormType.php
index 2ab818e..458e8c7 100644
--- a/src/AppBundle/Form/GenusFormType.php
+++ b/src/AppBundle/Form/GenusFormType.php
@@ -3,6 +3,7 @@
 namespace AppBundle\Form;
 
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
@@ -15,7 +16,12 @@ class GenusFormType extends AbstractType
             ->add('subFamily')
             ->add('speciesCount')
             ->add('funFact')
-            ->add('isPublished')
+            ->add('isPublished', ChoiceType::class, [
+                'choices' => [
+                    'Yes' => true,
+                    'No' => false,
+                ]
+            ])
             ->add('firstDiscoveredAt')
         ;
     }
