diff --git a/src/AppBundle/Form/ProductFormType.php b/src/AppBundle/Form/ProductFormType.php
index 4324ce9..1b5d270 100644
--- a/src/AppBundle/Form/ProductFormType.php
+++ b/src/AppBundle/Form/ProductFormType.php
@@ -25,7 +25,9 @@ class ProductFormType extends AbstractType
                     'No' => false,
                 ]
             ])
-            ->add('firstDiscoveredAt')
+            ->add('firstDiscoveredAt', null, [
+                'widget' => 'single_text'
+            ])
         ;
     }
 
