diff --git a/app/config/services.yml b/app/config/services.yml
index 1e00cc7b..d0d3780a 100644
--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -35,6 +35,8 @@ services:
         tags:
             - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\FormType }
 
+    AppBundle\Service\MessageGenerator: '@app.encouraging_message_generator'
+
     app.encouraging_message_generator:
         class: AppBundle\Service\MessageGenerator
         arguments:
diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/AppBundle/Controller/Admin/GenusAdminController.php
index 180567af..6bc292da 100644
--- a/src/AppBundle/Controller/Admin/GenusAdminController.php
+++ b/src/AppBundle/Controller/Admin/GenusAdminController.php
@@ -4,6 +4,7 @@ namespace AppBundle\Controller\Admin;
 
 use AppBundle\Entity\Genus;
 use AppBundle\Form\GenusFormType;
+use AppBundle\Service\MessageGenerator;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
@@ -61,7 +62,7 @@ class GenusAdminController extends Controller
     /**
      * @Route("/genus/{id}/edit", name="admin_genus_edit")
      */
-    public function editAction(Request $request, Genus $genus)
+    public function editAction(Request $request, Genus $genus, MessageGenerator $goodMessages)
     {
         $form = $this->createForm(GenusFormType::class, $genus);
 
@@ -76,7 +77,7 @@ class GenusAdminController extends Controller
 
             $this->addFlash(
                 'success',
-                $this->get('app.encouraging_message_generator')->getMessage()
+                $goodMessages->getMessage()
             );
 
             return $this->redirectToRoute('admin_genus_edit', [
