diff --git a/app/config/legacy_aliases.yml b/app/config/legacy_aliases.yml
deleted file mode 100644
index da7bb0d5..00000000
--- a/app/config/legacy_aliases.yml
+++ /dev/null
@@ -1,6 +0,0 @@
-services:
-    app.markdown_extension: '@AppBundle\Twig\MarkdownExtension'
-    app.security.login_form_authenticator: '@AppBundle\Security\LoginFormAuthenticator'
-    app.doctrine.hash_password_listener: '@AppBundle\Doctrine\HashPasswordListener'
-    app.form.help_form_extenion: '@AppBundle\Form\TypeExtension\HelpFormExtension'
-    app.encouraging_message_generator: '@AppBundle\Service\MessageGenerator'
diff --git a/app/config/security.yml b/app/config/security.yml
index b03a7bd4..011f26f4 100644
--- a/app/config/security.yml
+++ b/app/config/security.yml
@@ -22,7 +22,7 @@ security:
             anonymous: ~
             guard:
                 authenticators:
-                    - app.security.login_form_authenticator
+                    - AppBundle\Security\LoginFormAuthenticator
 
             logout:
                 path: /logout
diff --git a/app/config/services.yml b/app/config/services.yml
index 96bdb651..4842b671 100644
--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -1,8 +1,5 @@
 # Learn more about services, parameters and containers at
 # http://symfony.com/doc/current/book/service_container.html
-imports:
-    - { resource: legacy_aliases.yml }
-
 parameters:
 #    parameter_name: value
 
diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/AppBundle/Controller/Admin/GenusAdminController.php
index 180567af..ef974133 100644
--- a/src/AppBundle/Controller/Admin/GenusAdminController.php
+++ b/src/AppBundle/Controller/Admin/GenusAdminController.php
@@ -4,6 +4,7 @@ namespace AppBundle\Controller\Admin;
 
 use AppBundle\Entity\Genus;
 use AppBundle\Form\GenusFormType;
+use AppBundle\Service\MessageGenerator;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
@@ -61,7 +62,7 @@ class GenusAdminController extends Controller
     /**
      * @Route("/genus/{id}/edit", name="admin_genus_edit")
      */
-    public function editAction(Request $request, Genus $genus)
+    public function editAction(Request $request, Genus $genus, MessageGenerator $encourageMessages)
     {
         $form = $this->createForm(GenusFormType::class, $genus);
 
@@ -76,7 +77,7 @@ class GenusAdminController extends Controller
 
             $this->addFlash(
                 'success',
-                $this->get('app.encouraging_message_generator')->getMessage()
+                $encourageMessages->getMessage()
             );
 
             return $this->redirectToRoute('admin_genus_edit', [
diff --git a/src/AppBundle/Controller/UserController.php b/src/AppBundle/Controller/UserController.php
index da61629e..33758c93 100644
--- a/src/AppBundle/Controller/UserController.php
+++ b/src/AppBundle/Controller/UserController.php
@@ -5,6 +5,7 @@ namespace AppBundle\Controller;
 use AppBundle\Entity\User;
 use AppBundle\Form\UserEditForm;
 use AppBundle\Form\UserRegistrationForm;
+use AppBundle\Security\LoginFormAuthenticator;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
@@ -14,7 +15,7 @@ class UserController extends Controller
     /**
      * @Route("/register", name="user_register")
      */
-    public function registerAction(Request $request)
+    public function registerAction(Request $request, LoginFormAuthenticator $authenticator)
     {
         $form = $this->createForm(UserRegistrationForm::class);
 
@@ -32,7 +33,7 @@ class UserController extends Controller
                 ->authenticateUserAndHandleSuccess(
                     $user,
                     $request,
-                    $this->get('app.security.login_form_authenticator'),
+                    $authenticator,
                     'main'
                 );
         }
