diff --git a/src/AppBundle/DataFixtures/ORM/LoadFixtures.php b/src/AppBundle/DataFixtures/ORM/LoadFixtures.php
new file mode 100644
index 0000000..cb48a00
--- /dev/null
+++ b/src/AppBundle/DataFixtures/ORM/LoadFixtures.php
@@ -0,0 +1,21 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\Entity\Genus;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class LoadFixtures implements FixtureInterface
+{
+    public function load(ObjectManager $manager)
+    {
+        $genus = new Genus();
+        $genus->setName('Octopus'.rand(1, 100));
+        $genus->setSubFamily('Octopodinae');
+        $genus->setSpeciesCount(rand(100, 99999));
+
+        $manager->persist($genus);
+        $manager->flush();
+    }
+}
\ No newline at end of file
