diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index 7f49e54..eb849d8 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -1,12 +1 @@
-{{ form_start(genusForm) }}
-    {{ form_row(genusForm.name) }}
-    {{ form_row(genusForm.subFamily) }}
-    {{ form_row(genusForm.speciesCount, {
-        'label': 'Number of Species'
-    }) }}
-    {{ form_row(genusForm.funFact) }}
-    {{ form_row(genusForm.isPublished) }}
-    {{ form_row(genusForm.firstDiscoveredAt) }}
-
-    <button type="submit" class="btn btn-primary" formnovalidate>Save</button>
-{{ form_end(genusForm) }}
+{{ form(genusForm) }}
\ No newline at end of file
