diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index 01e3227..f87a511 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -8,7 +8,7 @@
         'attr': {
             'class': 'foo'
         },
-        'disabled': 'disabled',
+        'placeholder': 'Select a Subfamily'
     }) }}
     {{ form_row(genusForm.speciesCount, {
         'label': 'Number of Species'
