diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index 472ab76..3a375c4 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -1,3 +1,16 @@
+{% import _self as formMacros %}
+
+{% macro printGenusScientistRow(genusScientistForm) %}
+    <div class="col-xs-4 js-genus-scientist-item">
+        <a href="#" class="js-remove-scientist pull-right">
+            <span class="fa fa-close"></span>
+        </a>
+        {{ form_errors(genusScientistForm) }}
+        {{ form_row(genusScientistForm.user) }}
+        {{ form_row(genusScientistForm.yearsStudied) }}
+    </div>
+{% endmacro %}
+
 {{ form_start(genusForm) }}
     {{ form_errors(genusForm) }}
 
@@ -22,18 +35,11 @@
 
     <h3>Scientists</h3>
     <div class="row js-genus-scientist-wrapper"
-         data-prototype="{{ form_widget(genusForm.genusScientists.vars.prototype)|e('html_attr') }}"
+         data-prototype="{{ formMacros.printGenusScientistRow(genusForm.genusScientists.vars.prototype)|e('html_attr') }}"
          data-index="{{ genusForm.genusScientists|length }}"
     >
     {% for genusScientistForm in genusForm.genusScientists %}
-        <div class="col-xs-4 js-genus-scientist-item">
-            <a href="#" class="js-remove-scientist pull-right">
-                <span class="fa fa-close"></span>
-            </a>
-            {{ form_errors(genusScientistForm) }}
-            {{ form_row(genusScientistForm.user) }}
-            {{ form_row(genusScientistForm.yearsStudied) }}
-        </div>
+        {{ formMacros.printGenusScientistRow(genusScientistForm) }}
     {% endfor %}
         <a href="#" class="js-genus-scientist-add">
             <span class="fa fa-plus-circle"></span>
