diff --git a/app/Resources/views/user/edit.html.twig b/app/Resources/views/user/edit.html.twig
index 71bc247..5b39c82 100644
--- a/app/Resources/views/user/edit.html.twig
+++ b/app/Resources/views/user/edit.html.twig
@@ -14,7 +14,6 @@
                     {{ form_row(userForm.firstName) }}
                     {{ form_row(userForm.lastName) }}
                     {{ form_row(userForm.universityName) }}
-                    {{ form_row(userForm.studiedGenuses) }}
 
                     <button type="submit" class="btn btn-primary">Update</button>
                 {{ form_end(userForm) }}
diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 8043dbf..19d053a 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -219,24 +219,4 @@ class User implements UserInterface
     {
         return $this->studiedGenuses;
     }
-
-    public function addStudiedGenus(Genus $genus)
-    {
-        if ($this->studiedGenuses->contains($genus)) {
-            return;
-        }
-
-        $this->studiedGenuses[] = $genus;
-        $genus->addGenusScientist($this);
-    }
-
-    public function removeStudiedGenus(Genus $genus)
-    {
-        if (!$this->studiedGenuses->contains($genus)) {
-            return;
-        }
-
-        $this->studiedGenuses->removeElement($genus);
-        $genus->removeGenusScientist($this);
-    }
 }
diff --git a/src/AppBundle/Form/UserEditForm.php b/src/AppBundle/Form/UserEditForm.php
index b8f01eb..8829b50 100644
--- a/src/AppBundle/Form/UserEditForm.php
+++ b/src/AppBundle/Form/UserEditForm.php
@@ -22,13 +22,6 @@ class UserEditForm extends AbstractType
             ->add('firstName')
             ->add('lastName')
             ->add('universityName')
-            ->add('studiedGenuses', EntityType::class, [
-                'class' => Genus::class,
-                'multiple' => true,
-                'expanded' => true,
-                'choice_label' => 'name',
-                'by_reference' => false,
-            ])
         ;
     }
 
