diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index 5b5f3ec..be28466 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -1,9 +1,7 @@
 {{ form_start(genusForm) }}
     {{ form_errors(genusForm) }}
 
-    {{ form_label(genusForm.name) }}
-    {{ form_widget(genusForm.name) }}
-    {{ form_errors(genusForm.name) }}
+    {{ form_row(genusForm.name) }}
 
     {{ form_row(genusForm.subFamily) }}
     {{ form_row(genusForm.speciesCount, {
