diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/AppBundle/Controller/Admin/GenusAdminController.php
index 586e356..39bed81 100644
--- a/src/AppBundle/Controller/Admin/GenusAdminController.php
+++ b/src/AppBundle/Controller/Admin/GenusAdminController.php
@@ -36,7 +36,13 @@ class GenusAdminController extends Controller
         // only handles data on POST
         $form->handleRequest($request);
         if ($form->isSubmitted() && $form->isValid()) {
-            dump($form->getData());die;
+            $genus = $form->getData();
+
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($genus);
+            $em->flush();
+
+            return $this->redirectToRoute('admin_genus_list');
         }
 
         return $this->render('admin/genus/new.html.twig', [
