diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 4965fc8..7ca89ab 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -179,11 +179,15 @@ class Genus
         }
 
         $this->genusScientists[] = $user;
+        // not needed for persistence, just keeping both sides in sync
+        $user->addStudiedGenus($this);
     }
 
     public function removeGenusScientist(User $user)
     {
         $this->genusScientists->removeElement($user);
+        // not needed for persistence, just keeping both sides in sync
+        $user->removeStudiedGenus($this);
     }
 
     /**
