diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 205a5c8..12f762d 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -167,6 +167,10 @@ class Genus
 
     public function removeGenusScientist(User $user)
     {
+        if (!$this->genusScientists->contains($user)) {
+            return;
+        }
+
         $this->genusScientists->removeElement($user);
         // not needed for persistence, just keeping both sides in sync
         $user->removeStudiedGenus($this);
diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 698b396..b05cf68 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -233,6 +233,10 @@ class User implements UserInterface
 
     public function removeStudiedGenus(Genus $genus)
     {
+        if (!$this->studiedGenuses->contains($genus)) {
+            return;
+        }
+
         $this->studiedGenuses->removeElement($genus);
         $genus->addGenusScientist($this);
     }
