diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index f45c1af..50c2c9a 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -228,10 +228,12 @@ class User implements UserInterface
         }
 
         $this->studiedGenuses[] = $genus;
+        $genus->addGenusScientist($this);
     }
 
     public function removeStudiedGenus(Genus $genus)
     {
         $this->studiedGenuses->removeElement($genus);
+        $genus->removeGenusScientist($this);
     }
 }
