diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 7ca89ab..259f24a 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -185,6 +185,10 @@ class Genus
 
     public function removeGenusScientist(User $user)
     {
+        if (!$this->genusScientists->contains($user)) {
+            return;
+        }
+
         $this->genusScientists->removeElement($user);
         // not needed for persistence, just keeping both sides in sync
         $user->removeStudiedGenus($this);
diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 50c2c9a..85f5458 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -233,6 +233,10 @@ class User implements UserInterface
 
     public function removeStudiedGenus(Genus $genus)
     {
+        if (!$this->studiedGenuses->contains($genus)) {
+            return;
+        }
+
         $this->studiedGenuses->removeElement($genus);
         $genus->removeGenusScientist($this);
     }
