diff --git a/src/AppBundle/Repository/GenusNoteRepository.php b/src/AppBundle/Repository/GenusNoteRepository.php
index ae393f6..9bde0e5 100644
--- a/src/AppBundle/Repository/GenusNoteRepository.php
+++ b/src/AppBundle/Repository/GenusNoteRepository.php
@@ -15,6 +15,11 @@ class GenusNoteRepository extends EntityRepository
     public function findAllRecentNotesForGenus(Genus $genus)
     {
         return $this->createQueryBuilder('genus_note')
+            ->andWhere('genus_note.genus = :genus')
+            ->setParameter('genus', $genus)
+            ->andWhere('genus_note.createdAt > :recentDate')
+            ->setParameter('recentDate', new \DateTime('-3 months'))
+            ->orderBy('genus_note.createdAt', 'DESC')
             ->getQuery()
             ->execute();
     }
