diff --git a/src/AppBundle/Entity/GenusScientist.php b/src/AppBundle/Entity/GenusScientist.php
new file mode 100644
index 0000000..172415c
--- /dev/null
+++ b/src/AppBundle/Entity/GenusScientist.php
@@ -0,0 +1,71 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="genus_scientist")
+ */
+class GenusScientist
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Genus")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $genus;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="User")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private $user;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $yearsStudied;
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function getGenus()
+    {
+        return $this->genus;
+    }
+
+    public function setGenus($genus)
+    {
+        $this->genus = $genus;
+    }
+
+    public function getUser()
+    {
+        return $this->user;
+    }
+
+    public function setUser($user)
+    {
+        $this->user = $user;
+    }
+
+    public function getYearsStudied()
+    {
+        return $this->yearsStudied;
+    }
+
+    public function setYearsStudied($yearsStudied)
+    {
+        $this->yearsStudied = $yearsStudied;
+    }
+}
\ No newline at end of file
