diff --git a/app/Resources/views/genus/show.html.twig b/app/Resources/views/genus/show.html.twig
index 0055569..6ec1869 100644
--- a/app/Resources/views/genus/show.html.twig
+++ b/app/Resources/views/genus/show.html.twig
@@ -24,9 +24,10 @@
                         {% for genusScientist in genus.genusScientists %}
                             <li class="list-group-item js-scientist-item">
                                 <a href="{{ path('user_show', {
-                                    'id': genusScientist.id
+                                    'id': genusScientist.user.id
                                 }) }}">
-                                    {{ genusScientist.fullName }}
+                                    {{ genusScientist.user.fullName }}
+                                    ({{ genusScientist.yearsStudied }} years)
                                 </a>
 
                                 <a href="#"
diff --git a/app/Resources/views/user/show.html.twig b/app/Resources/views/user/show.html.twig
index 21382a1..1668c2f 100644
--- a/app/Resources/views/user/show.html.twig
+++ b/app/Resources/views/user/show.html.twig
@@ -39,12 +39,12 @@
             <div class="col-xs-4">
                 <h3>Genus Studied</h3>
                 <ul class="list-group">
-                    {% for genusStudied in user.studiedGenuses %}
+                    {% for genusScientist in user.studiedGenuses %}
                         <li class="list-group-item">
                             <a href="{{ path('genus_show', {
-                                'slug': genusStudied.slug
+                                'slug': genusScientist.genus.slug
                             }) }}">
-                                {{ genusStudied.name }}
+                                {{ genusScientist.genus.name }}
                             </a>
                         </li>
                     {% endfor %}
diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index fe1ca68..c5e5b94 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -4,6 +4,7 @@ namespace AppBundle\Controller;
 
 use AppBundle\Entity\Genus;
 use AppBundle\Entity\GenusNote;
+use AppBundle\Entity\GenusScientist;
 use AppBundle\Service\MarkdownTransformer;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
@@ -38,8 +39,12 @@ class GenusController extends Controller
 
         $user = $em->getRepository('AppBundle:User')
             ->findOneBy(['email' => 'aquanaut1@example.org']);
-        $genus->addGenusScientist($user);
-        $genus->addGenusScientist($user); // duplicate is ignored!
+
+        $genusScientist = new GenusScientist();
+        $genusScientist->setGenus($genus);
+        $genusScientist->setUser($user);
+        $genusScientist->setYearsStudied(10);
+        $em->persist($genusScientist);
 
         $em->persist($genus);
         $em->persist($genusNote);
diff --git a/src/AppBundle/DataFixtures/ORM/fixtures.yml b/src/AppBundle/DataFixtures/ORM/fixtures.yml
index d7b9890..22f9f40 100644
--- a/src/AppBundle/DataFixtures/ORM/fixtures.yml
+++ b/src/AppBundle/DataFixtures/ORM/fixtures.yml
@@ -6,7 +6,7 @@ AppBundle\Entity\Genus:
         funFact: <sentence()>
         isPublished: <boolean(75)>
         firstDiscoveredAt: <dateTimeBetween('-200 years', 'now')>
-        genusScientists: '3x @user.aquanaut_*'
+        # genusScientists: '3x @user.aquanaut_*'
 
 AppBundle\Entity\GenusNote:
     genus.note_{1..100}:
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index bb07471..ba5c87d 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -194,7 +194,7 @@ class Genus
     }
 
     /**
-     * @return ArrayCollection|User[]
+     * @return ArrayCollection|GenusScientist[]
      */
     public function getGenusScientists()
     {
diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 1f00eb0..8043dbf 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -213,7 +213,7 @@ class User implements UserInterface
     }
 
     /**
-     * @return ArrayCollection|Genus[]
+     * @return ArrayCollection|GenusScientist[]
      */
     public function getStudiedGenuses()
     {
