diff --git a/app/DoctrineMigrations/Version20161017160251.php b/app/DoctrineMigrations/Version20161017160251.php
index 300b10a..8cb53b9 100644
--- a/app/DoctrineMigrations/Version20161017160251.php
+++ b/app/DoctrineMigrations/Version20161017160251.php
@@ -21,10 +21,9 @@ class Version20161017160251 extends AbstractMigration
         $this->addSql('ALTER TABLE genus_scientist DROP FOREIGN KEY FK_66CF3FA885C4074C');
         $this->addSql('ALTER TABLE genus_scientist DROP FOREIGN KEY FK_66CF3FA8A76ED395');
         $this->addSql('ALTER TABLE genus_scientist DROP PRIMARY KEY');
-        $this->addSql('ALTER TABLE genus_scientist ADD id INT AUTO_INCREMENT NOT NULL, ADD years_studied VARCHAR(255) NOT NULL');
+        $this->addSql('ALTER TABLE genus_scientist ADD id INT AUTO_INCREMENT NOT NULL, ADD PRIMARY KEY (id), ADD years_studied VARCHAR(255) NOT NULL');
         $this->addSql('ALTER TABLE genus_scientist ADD CONSTRAINT FK_66CF3FA885C4074C FOREIGN KEY (genus_id) REFERENCES genus (id)');
         $this->addSql('ALTER TABLE genus_scientist ADD CONSTRAINT FK_66CF3FA8A76ED395 FOREIGN KEY (user_id) REFERENCES user (id)');
-        $this->addSql('ALTER TABLE genus_scientist ADD PRIMARY KEY (id)');
     }
 
     /**
