diff --git a/app/Resources/views/genus/show.html.twig b/app/Resources/views/genus/show.html.twig
index a281f31..dd09efe 100644
--- a/app/Resources/views/genus/show.html.twig
+++ b/app/Resources/views/genus/show.html.twig
@@ -25,17 +25,25 @@
         <div><i class="fa fa-plus plus-btn"></i></div>
     </div>
     <section id="cd-timeline">
-        {#{% for note in notes %}#}
-            {#<div class="cd-timeline-block">#}
-                {#<div class="cd-timeline-img">#}
-        			{#<img src="{{ asset('images/leanna.jpeg') }}" class="img-circle" alt="Leanna!">#}
-        		{#</div>#}
-                {#<div class="cd-timeline-content">#}
-        			{#<h2><a href="#">AquaPelham</a></h2>#}
-        			{#<p>{{ note }}</p>#}
-        			{#<span class="cd-date">Dec. 10, 2015</span>#}
-        		{#</div>#}
-            {#</div>#}
-        {#{% endfor %}#}
+        <ul class="js-notes-wrapper">
+        </ul>
     </section>
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script>
+        $(document).ready(function() {
+            $.ajax({
+                url: '{{ path('genus_show_notes', {'genusName': name}) }}',
+                method: 'GET',
+                success: function(data) {
+                    var $notesWrapper = $('.js-notes-wrapper');
+                    $.each(data.notes, function(key, note) {
+                        $notesWrapper.append('<li>'+note+'</li>');
+                    });
+                }
+            });
+        });
+    </script>
+{% endblock %}
