diff --git a/app/Resources/views/genus/show.html.twig b/app/Resources/views/genus/show.html.twig
index 62b4c38..b011382 100644
--- a/app/Resources/views/genus/show.html.twig
+++ b/app/Resources/views/genus/show.html.twig
@@ -29,8 +29,10 @@
 
     <script type="text/babel" src="{{ asset('js/notes.react.js') }}"></script>
     <script type="text/babel">
+        var notesUrl = '{{ path('genus_show_notes', {'genusName': name}) }}';
+
         ReactDOM.render(
-          <NoteSection />,
+          <NoteSection url={notesUrl} />,
           document.getElementById('js-notes-wrapper')
         );
     </script>
diff --git a/web/js/notes.react.js b/web/js/notes.react.js
index 0735701..c88f934 100644
--- a/web/js/notes.react.js
+++ b/web/js/notes.react.js
@@ -12,7 +12,7 @@ var NoteSection = React.createClass({
 
     loadNotesFromServer: function() {
         $.ajax({
-            url: '/genus/octopus/notes',
+            url: this.props.url,
             success: function (data) {
                 this.setState({notes: data.notes});
             }.bind(this)
