diff --git a/app/Resources/views/genus/list.html.twig b/app/Resources/views/genus/list.html.twig
index 9e7e36e..eb8b2c8 100644
--- a/app/Resources/views/genus/list.html.twig
+++ b/app/Resources/views/genus/list.html.twig
@@ -12,7 +12,11 @@
         <tbody>
             {% for genus in genuses %}
                 <tr>
-                    <td>{{ genus.name }}</td>
+                    <td>
+                        <a href="{{ path('genus_show', {'genusName': genus.name}) }}">
+                            {{ genus.name }}
+                        </a>
+                    </td>
                     <td>{{ genus.speciesCount }}</td>
                     <td>{{ genus.updatedAt|date('Y-m-d') }}</td>
                 </tr>
diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index efd665c..32cb716 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -44,7 +44,7 @@ class GenusController extends Controller
     }
 
     /**
-     * @Route("/genus/{genusName}")
+     * @Route("/genus/{genusName}", name="genus_show")
      */
     public function showAction($genusName)
     {
