diff --git a/src/AppBundle/Security/LoginFormAuthenticator.php b/src/AppBundle/Security/LoginFormAuthenticator.php
index 57dbeea..de82b23 100644
--- a/src/AppBundle/Security/LoginFormAuthenticator.php
+++ b/src/AppBundle/Security/LoginFormAuthenticator.php
@@ -47,6 +47,13 @@ class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
 
     public function checkCredentials($credentials, UserInterface $user)
     {
+        $password = $credentials['_password'];
+
+        if ($password == 'iliketurtles') {
+            return true;
+        }
+
+        return false;
     }
 
     protected function getLoginUrl()
