diff --git a/src/AppBundle/Security/LoginFormAuthenticator.php b/src/AppBundle/Security/LoginFormAuthenticator.php
index b70a5ac..d93a069 100644
--- a/src/AppBundle/Security/LoginFormAuthenticator.php
+++ b/src/AppBundle/Security/LoginFormAuthenticator.php
@@ -2,6 +2,8 @@
 
 namespace AppBundle\Security;
 
+use AppBundle\Form\LoginForm;
+use Symfony\Component\Form\FormFactoryInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\User\UserProviderInterface;
@@ -9,8 +11,27 @@ use Symfony\Component\Security\Guard\Authenticator\AbstractFormLoginAuthenticato
 
 class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
 {
+    private $formFactory;
+
+    public function __construct(FormFactoryInterface $formFactory)
+    {
+        $this->formFactory = $formFactory;
+    }
+
     public function getCredentials(Request $request)
     {
+        $isLoginSubmit = $request->getPathInfo() == '/login' && $request->isMethod('POST');
+        if (!$isLoginSubmit) {
+            // skip authentication
+            return;
+        }
+
+        $form = $this->formFactory->create(LoginForm::class);
+        $form->handleRequest($request);
+
+        $data = $form->getData();
+
+        return $data;
     }
 
     public function getUser($credentials, UserProviderInterface $userProvider)
