diff --git a/src/AppBundle/Security/LoginFormAuthenticator.php b/src/AppBundle/Security/LoginFormAuthenticator.php
index 3f71953..52b49ad 100644
--- a/src/AppBundle/Security/LoginFormAuthenticator.php
+++ b/src/AppBundle/Security/LoginFormAuthenticator.php
@@ -45,6 +45,13 @@ class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
 
     public function checkCredentials($credentials, UserInterface $user)
     {
+        $password = $credentials['_password'];
+
+        if ($password == 'iliketurtles') {
+            return true;
+        }
+
+        return false;
     }
 
     protected function getLoginUrl()
