diff --git a/src/AppBundle/Security/LoginFormAuthenticator.php b/src/AppBundle/Security/LoginFormAuthenticator.php
index a1c0062..48065b9 100644
--- a/src/AppBundle/Security/LoginFormAuthenticator.php
+++ b/src/AppBundle/Security/LoginFormAuthenticator.php
@@ -2,6 +2,8 @@
 
 namespace AppBundle\Security;
 
+use AppBundle\Form\LoginForm;
+use Symfony\Component\Form\FormFactoryInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
@@ -11,8 +13,25 @@ use Symfony\Component\Security\Guard\AbstractGuardAuthenticator;
 
 class LoginFormAuthenticator extends AbstractGuardAuthenticator
 {
+    private $formFactory;
+
+    public function __construct(FormFactoryInterface $formFactory)
+    {
+        $this->formFactory = $formFactory;
+    }
+
     public function getCredentials(Request $request)
     {
+        $isLoginSubmit = $request->getPathInfo() == '/login' && $request->isMethod('POST');
+        if (!$isLoginSubmit) {
+            // skip authentication
+            return;
+        }
+
+        $form = $this->formFactory->create(LoginForm::class);
+        $form->handleRequest($request);
+
+        return $form->getData();
     }
 
     public function getUser($credentials, UserProviderInterface $userProvider)
