diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index d92aafc..1eb2498 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -61,9 +61,15 @@ class Genus
      */
     private $notes;
 
+    /**
+     * @ORM\ManyToMany(targetEntity="User")
+     */
+    private $scientistUsers;
+
     public function __construct()
     {
         $this->notes = new ArrayCollection();
+        $this->scientistUsers = new ArrayCollection();
     }
 
     public function getId()
