diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index ace2f64..c99e925 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -36,6 +36,10 @@ class GenusController extends Controller
         $genusNote->setCreatedAt(new \DateTime('-1 month'));
         $genusNote->setGenus($genus);
 
+        $user = $em->getRepository('AppBundle:User')
+            ->findOneBy(['email' => 'aquanaut1@example.org']);
+        $genus->addGenusScientist($user);
+
         $em->persist($genus);
         $em->persist($genusNote);
         $em->flush();
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 0e06a2c..07e1c25 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -171,4 +171,9 @@ class Genus
     {
         $this->slug = $slug;
     }
+
+    public function addGenusScientist(User $user)
+    {
+        $this->genusScientists[] = $user;
+    }
 }
