diff --git a/src/AppBundle/Doctrine/HashPasswordListener.php b/src/AppBundle/Doctrine/HashPasswordListener.php
index 43228c5..49a15b8 100644
--- a/src/AppBundle/Doctrine/HashPasswordListener.php
+++ b/src/AppBundle/Doctrine/HashPasswordListener.php
@@ -2,11 +2,34 @@
 
 namespace AppBundle\Doctrine;
 
+use AppBundle\Entity\User;
 use Doctrine\Common\EventSubscriber;
 use Doctrine\ORM\Event\LifecycleEventArgs;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoder;
 
 class HashPasswordListener implements EventSubscriber
 {
+    private $passwordEncoder;
+
+    public function __construct(UserPasswordEncoder $passwordEncoder)
+    {
+        $this->passwordEncoder = $passwordEncoder;
+    }
+
+    public function prePersist(LifecycleEventArgs $args)
+    {
+        $entity = $args->getEntity();
+        if (!$entity instanceof User) {
+            return;
+        }
+
+        $encoded = $this->passwordEncoder->encodePassword(
+            $entity,
+            $entity->getPlainPassword()
+        );
+        $entity->setPassword($encoded);
+    }
+
     public function getSubscribedEvents()
     {
         return ['prePersist', 'preUpdate'];
