diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 31672b9..25c56da 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -61,7 +61,7 @@ class User implements UserInterface
 
     public function eraseCredentials()
     {
-        // leaving blank - I don't need/have a password!
+        $this->plainPassword = null;
     }
 
     public function getEmail()
