diff --git a/src/AppBundle/Doctrine/HashPasswordListener.php b/src/AppBundle/Doctrine/HashPasswordListener.php
new file mode 100644
index 0000000..43228c5
--- /dev/null
+++ b/src/AppBundle/Doctrine/HashPasswordListener.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace AppBundle\Doctrine;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\LifecycleEventArgs;
+
+class HashPasswordListener implements EventSubscriber
+{
+    public function getSubscribedEvents()
+    {
+        return ['prePersist', 'preUpdate'];
+    }
+}
