diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index cda6096..bbb2988 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -31,6 +31,13 @@ class User implements UserInterface
      */
     private $password;
 
+    /**
+     * A non-persisted field that's used to create the encoded password.
+     *
+     * @var string
+     */
+    private $plainPassword;
+
     // needed by the security system
     public function getUsername()
     {
@@ -71,4 +78,14 @@ class User implements UserInterface
     {
         $this->password = $password;
     }
+
+    public function getPlainPassword()
+    {
+        return $this->plainPassword;
+    }
+
+    public function setPlainPassword($plainPassword)
+    {
+        $this->plainPassword = $plainPassword;
+    }
 }
