diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index fc4d593..f5663e4 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -35,7 +35,7 @@ class GenusController extends Controller
     {
         $em = $this->getDoctrine()->getManager();
 
-        $genuses = $em->getRepository('AppBundle\Entity\Genus')
+        $genuses = $em->getRepository('AppBundle:Genus')
             ->findAll();
         dump($genuses);die;
     }
