diff --git a/app/Resources/views/user/register.html.twig b/app/Resources/views/user/register.html.twig
index db1c1a8..5f4fc42 100644
--- a/app/Resources/views/user/register.html.twig
+++ b/app/Resources/views/user/register.html.twig
@@ -8,8 +8,12 @@
 
                 {{ form_start(form) }}
                     {{ form_row(form.email) }}
-                    {{ form_row(form.plainPassword.first) }}
-                    {{ form_row(form.plainPassword.second) }}
+                    {{ form_row(form.plainPassword.first, {
+                        'label': 'Password'
+                    }) }}
+                    {{ form_row(form.plainPassword.second, {
+                        'label': 'Repeat Password'
+                    }) }}
 
                     <button type="submit" class="btn btn-primary" formnovalidate>Register</button>
                 {{ form_end(form) }}
