diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 50f3643..32876ae 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -128,5 +128,26 @@ class GenusController extends Controller
      */
     public function removeScientistUserAction($genusName, $scientistId)
     {
+        $em = $this->getDoctrine()->getManager();
+
+        $genus = $em->getRepository('AppBundle:Genus')
+            ->findOneBy(['name' => $genusName]);
+
+        if (!$genus) {
+            throw $this->createNotFoundException('genus not found');
+        }
+
+        $scientistUser = $em->getRepository('AppBundle:User')
+            ->find($scientistId);
+
+        if (!$scientistUser) {
+            throw $this->createNotFoundException('scientist not found');
+        }
+
+        $genus->removeScientistUser($scientistUser);
+        $em->persist($genus);
+        $em->flush();
+
+        return new Response(null, 204);
     }
 }
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 80b8146..b5bb33e 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -163,6 +163,11 @@ class Genus
         $this->scientistUsers[] = $user;
     }
 
+    public function removeScientistUser(User $user)
+    {
+        $this->scientistUsers->removeElement($user);
+    }
+
     /**
      * @return ArrayCollection|User[]
      */
