diff --git a/app/Resources/views/admin/genus/new.html.twig b/app/Resources/views/admin/genus/new.html.twig
index 9672b80..0aaef8e 100644
--- a/app/Resources/views/admin/genus/new.html.twig
+++ b/app/Resources/views/admin/genus/new.html.twig
@@ -27,11 +27,16 @@
                 <h1>New Genus</h1>
 
                 {{ form_start(genusForm) }}
-                    {{ form_widget(genusForm) }}
+                    {{ form_row(genusForm.name) }}
+                    {{ form_row(genusForm.subFamily) }}
+                    {{ form_row(genusForm.speciesCount) }}
+                    {{ form_row(genusForm.funFact) }}
+                    {{ form_row(genusForm.isPublished) }}
+                    {{ form_row(genusForm.firstDiscoveredAt) }}
 
                     <button type="submit" class="btn btn-primary">Save</button>
                 {{ form_end(genusForm) }}
             </div>
         </div>
     </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
