diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 199046a..7013718 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -35,6 +35,7 @@ class GenusController extends Controller
     {
         $em = $this->getDoctrine()->getManager();
 
+        dump($em->getRepository('AppBundle:Genus'));
         $genuses = $em->getRepository('AppBundle:Genus')
             ->findAll();
 
