diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
new file mode 100644
index 0000000..a6a1917
--- /dev/null
+++ b/src/AppBundle/Controller/GenusController.php
@@ -0,0 +1,17 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Component\HttpFoundation\Response;
+
+class GenusController
+{
+    /**
+     * @Route("/genus")
+     */
+    public function showAction()
+    {
+        return new Response('Under the sea!');
+    }
+}
