diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 6c150aa..9c7b7c8 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -23,7 +23,7 @@ class GenusController extends Controller
         $em->persist($genus);
         $em->flush();
 
-        return new Response('Genus created!');
+        return new Response('<html><body>Genus created!</body></html>');
     }
 
     /**
