diff --git a/src/AppBundle/Service/MarkdownTransformer.php b/src/AppBundle/Service/MarkdownTransformer.php
index 51a7c43..fb3bffb 100644
--- a/src/AppBundle/Service/MarkdownTransformer.php
+++ b/src/AppBundle/Service/MarkdownTransformer.php
@@ -2,11 +2,13 @@
 
 namespace AppBundle\Service;
 
+use Knp\Bundle\MarkdownBundle\MarkdownParserInterface;
+
 class MarkdownTransformer
 {
     private $markdownParser;
 
-    public function __construct($markdownParser)
+    public function __construct(MarkdownParserInterface $markdownParser)
     {
         $this->markdownParser = $markdownParser;
     }
@@ -14,6 +16,6 @@ class MarkdownTransformer
     public function parse($str)
     {
         return $this->markdownParser
-            ->transform($str);
+            ->transformMarkdown($str);
     }
 }
