diff --git a/tests/AppBundle/Controller/DefaultControllerTest.php b/tests/AppBundle/Controller/DefaultControllerTest.php
deleted file mode 100644
index 594803c..0000000
--- a/tests/AppBundle/Controller/DefaultControllerTest.php
+++ /dev/null
@@ -1,18 +0,0 @@
-<?php
-
-namespace Tests\AppBundle\Controller;
-
-use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
-
-class DefaultControllerTest extends WebTestCase
-{
-    public function testIndex()
-    {
-        $client = static::createClient();
-
-        $crawler = $client->request('GET', '/');
-
-        $this->assertEquals(200, $client->getResponse()->getStatusCode());
-        $this->assertContains('Welcome to Symfony', $crawler->filter('#container h1')->text());
-    }
-}
diff --git a/tests/Controller/DefaultControllerTest.php b/tests/Controller/DefaultControllerTest.php
new file mode 100644
index 0000000..9172986
--- /dev/null
+++ b/tests/Controller/DefaultControllerTest.php
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/');
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+        $this->assertContains('Welcome to Symfony', $crawler->filter('#container h1')->text());
+    }
+}
