diff --git a/src/AppBundle/AppBundle.php b/src/AppBundle.php
similarity index 81%
rename from src/AppBundle/AppBundle.php
rename to src/AppBundle.php
index 05123b6..9251095 100644
--- a/src/AppBundle/AppBundle.php
+++ b/src/AppBundle.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle;
+namespace App;
 
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
diff --git a/src/AppBundle/DataFixtures/ORM/fixtures.yml b/src/AppBundle/DataFixtures/ORM/fixtures.yml
deleted file mode 100644
index 12c2484..0000000
--- a/src/AppBundle/DataFixtures/ORM/fixtures.yml
+++ /dev/null
@@ -1,43 +0,0 @@
-AppBundle\Entity\Genus:
-    genus_{1..10}:
-        name: <genus()>
-        subFamily: '@subfamily_*'
-        speciesCount: <numberBetween(100, 100000)>
-        funFact: <sentence()>
-        isPublished: <boolean(75)>
-        firstDiscoveredAt: <dateTimeBetween('-200 years', 'now')>
-        # genusScientists: '3x @user.aquanaut_*'
-
-AppBundle\Entity\GenusNote:
-    genus.note_{1..100}:
-        username: <userName()>
-        userAvatarFilename: '50%? leanna.jpeg : ryan.jpeg'
-        note: <paragraph()>
-        createdAt: <dateTimeBetween('-6 months', 'now')>
-        genus: '@genus_*'
-
-AppBundle\Entity\SubFamily:
-    subfamily_{1..10}:
-        # use some fake data for this
-        name: <lastName()>
-
-AppBundle\Entity\User:
-    user_{1..10}:
-        email: weaverryan+<current()>@gmail.com
-        plainPassword: iliketurtles
-        roles: ['ROLE_ADMIN']
-        avatarUri: <imageUrl(100, 100, 'abstract')>
-    user.aquanaut_{1..10}:
-        email: aquanaut<current()>@example.org
-        plainPassword: aquanote
-        isScientist: true
-        firstName: <firstName()>
-        lastName: <lastName()>
-        universityName: <company()> University
-        avatarUri: <imageUrl(100, 100, 'abstract')>
-
-AppBundle\Entity\GenusScientist:
-    genus.scientist_{1..50}:
-        user: '@user.aquanaut_*'
-        genus: '@genus_*'
-        yearsStudied: <numberBetween(1, 30)>
diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/Controller/Admin/GenusAdminController.php
similarity index 94%
rename from src/AppBundle/Controller/Admin/GenusAdminController.php
rename to src/Controller/Admin/GenusAdminController.php
index 2895fce..6365d74 100644
--- a/src/AppBundle/Controller/Admin/GenusAdminController.php
+++ b/src/Controller/Admin/GenusAdminController.php
@@ -1,10 +1,10 @@
 <?php
 
-namespace AppBundle\Controller\Admin;
+namespace App\Controller\Admin;
 
-use AppBundle\Entity\Genus;
-use AppBundle\Form\GenusFormType;
-use AppBundle\Service\MessageManager;
+use App\Entity\Genus;
+use App\Form\GenusFormType;
+use App\Service\MessageManager;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
diff --git a/src/AppBundle/Controller/GenusController.php b/src/Controller/GenusController.php
similarity index 95%
rename from src/AppBundle/Controller/GenusController.php
rename to src/Controller/GenusController.php
index ab349e5..4c0a852 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/Controller/GenusController.php
@@ -1,11 +1,11 @@
 <?php
 
-namespace AppBundle\Controller;
+namespace App\Controller;
 
-use AppBundle\Entity\Genus;
-use AppBundle\Entity\GenusNote;
-use AppBundle\Entity\GenusScientist;
-use AppBundle\Service\MarkdownTransformer;
+use App\Entity\Genus;
+use App\Entity\GenusNote;
+use App\Entity\GenusScientist;
+use App\Service\MarkdownTransformer;
 use Psr\Log\LoggerInterface;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
diff --git a/src/AppBundle/Controller/MainController.php b/src/Controller/MainController.php
similarity index 86%
rename from src/AppBundle/Controller/MainController.php
rename to src/Controller/MainController.php
index ae8b6ea..31fc4ca 100644
--- a/src/AppBundle/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Controller;
+namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
diff --git a/src/AppBundle/Controller/SecurityController.php b/src/Controller/SecurityController.php
similarity index 94%
rename from src/AppBundle/Controller/SecurityController.php
rename to src/Controller/SecurityController.php
index d021ccf..6d46cc9 100644
--- a/src/AppBundle/Controller/SecurityController.php
+++ b/src/Controller/SecurityController.php
@@ -1,10 +1,10 @@
 <?php
 
-namespace AppBundle\Controller;
+namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use AppBundle\Form\LoginForm;
+use App\Form\LoginForm;
 use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
 
 class SecurityController extends Controller
diff --git a/src/AppBundle/Controller/UserController.php b/src/Controller/UserController.php
similarity index 92%
rename from src/AppBundle/Controller/UserController.php
rename to src/Controller/UserController.php
index a75c404..810f3b0 100644
--- a/src/AppBundle/Controller/UserController.php
+++ b/src/Controller/UserController.php
@@ -1,11 +1,11 @@
 <?php
 
-namespace AppBundle\Controller;
+namespace App\Controller;
 
-use AppBundle\Entity\User;
-use AppBundle\Form\UserEditForm;
-use AppBundle\Form\UserRegistrationForm;
-use AppBundle\Security\LoginFormAuthenticator;
+use App\Entity\User;
+use App\Form\UserEditForm;
+use App\Form\UserRegistrationForm;
+use App\Security\LoginFormAuthenticator;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Request;
diff --git a/src/AppBundle/DataFixtures/ORM/LoadFixtures.php b/src/DataFixtures/ORM/LoadFixtures.php
similarity index 93%
rename from src/AppBundle/DataFixtures/ORM/LoadFixtures.php
rename to src/DataFixtures/ORM/LoadFixtures.php
index 247253c..4db98ab 100644
--- a/src/AppBundle/DataFixtures/ORM/LoadFixtures.php
+++ b/src/DataFixtures/ORM/LoadFixtures.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\DataFixtures\ORM;
+namespace App\DataFixtures\ORM;
 
-use AppBundle\Entity\Genus;
+use App\Entity\Genus;
 use Doctrine\Common\DataFixtures\FixtureInterface;
 use Doctrine\Common\Persistence\ObjectManager;
 use Nelmio\Alice\Fixtures;
diff --git a/src/AppBundle/Doctrine/HashPasswordListener.php b/src/Doctrine/HashPasswordListener.php
similarity index 96%
rename from src/AppBundle/Doctrine/HashPasswordListener.php
rename to src/Doctrine/HashPasswordListener.php
index a0facae..d5abfcb 100644
--- a/src/AppBundle/Doctrine/HashPasswordListener.php
+++ b/src/Doctrine/HashPasswordListener.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Doctrine;
+namespace App\Doctrine;
 
-use AppBundle\Entity\User;
+use App\Entity\User;
 use Doctrine\Common\EventSubscriber;
 use Doctrine\ORM\Event\LifecycleEventArgs;
 use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
diff --git a/src/AppBundle/Entity/Genus.php b/src/Entity/Genus.php
similarity index 98%
rename from src/AppBundle/Entity/Genus.php
rename to src/Entity/Genus.php
index 9d95676..f4f6a06 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/Entity/Genus.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
-use AppBundle\Repository\GenusRepository;
+use App\Repository\GenusRepository;
 use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\Mapping as ORM;
diff --git a/src/AppBundle/Entity/GenusNote.php b/src/Entity/GenusNote.php
similarity index 98%
rename from src/AppBundle/Entity/GenusNote.php
rename to src/Entity/GenusNote.php
index 04387e8..c5d097f 100644
--- a/src/AppBundle/Entity/GenusNote.php
+++ b/src/Entity/GenusNote.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 
diff --git a/src/AppBundle/Entity/GenusScientist.php b/src/Entity/GenusScientist.php
similarity index 98%
rename from src/AppBundle/Entity/GenusScientist.php
rename to src/Entity/GenusScientist.php
index d392492..61dd497 100644
--- a/src/AppBundle/Entity/GenusScientist.php
+++ b/src/Entity/GenusScientist.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
diff --git a/src/AppBundle/Entity/SubFamily.php b/src/Entity/SubFamily.php
similarity index 95%
rename from src/AppBundle/Entity/SubFamily.php
rename to src/Entity/SubFamily.php
index 95e9de7..7bcf4e6 100644
--- a/src/AppBundle/Entity/SubFamily.php
+++ b/src/Entity/SubFamily.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\ORM\Mapping as ORM;
 
diff --git a/src/AppBundle/Entity/User.php b/src/Entity/User.php
similarity index 99%
rename from src/AppBundle/Entity/User.php
rename to src/Entity/User.php
index 19d053a..96b6168 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/Entity/User.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\Common\Collections\ArrayCollection;
 use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
diff --git a/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php b/src/EventSubscriber/AddNiceHeaderEventSubscriber.php
similarity index 94%
rename from src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
rename to src/EventSubscriber/AddNiceHeaderEventSubscriber.php
index fe74265..138ba39 100644
--- a/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
+++ b/src/EventSubscriber/AddNiceHeaderEventSubscriber.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\EventSubscriber;
+namespace App\EventSubscriber;
 
-use AppBundle\Service\MessageManager;
+use App\Service\MessageManager;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
diff --git a/src/AppBundle/Form/GenusFormType.php b/src/Form/GenusFormType.php
similarity index 95%
rename from src/AppBundle/Form/GenusFormType.php
rename to src/Form/GenusFormType.php
index cdeb7b7..7ab57f5 100644
--- a/src/AppBundle/Form/GenusFormType.php
+++ b/src/Form/GenusFormType.php
@@ -1,11 +1,11 @@
 <?php
 
-namespace AppBundle\Form;
+namespace App\Form;
 
-use AppBundle\Entity\SubFamily;
-use AppBundle\Entity\User;
-use AppBundle\Repository\SubFamilyRepository;
-use AppBundle\Repository\UserRepository;
+use App\Entity\SubFamily;
+use App\Entity\User;
+use App\Repository\SubFamilyRepository;
+use App\Repository\UserRepository;
 use Symfony\Bridge\Doctrine\Form\Type\EntityType;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
diff --git a/src/AppBundle/Form/GenusScientistEmbeddedForm.php b/src/Form/GenusScientistEmbeddedForm.php
similarity index 90%
rename from src/AppBundle/Form/GenusScientistEmbeddedForm.php
rename to src/Form/GenusScientistEmbeddedForm.php
index f6232dd..421dacd 100644
--- a/src/AppBundle/Form/GenusScientistEmbeddedForm.php
+++ b/src/Form/GenusScientistEmbeddedForm.php
@@ -1,10 +1,10 @@
 <?php
 
-namespace AppBundle\Form;
+namespace App\Form;
 
-use AppBundle\Entity\GenusScientist;
-use AppBundle\Entity\User;
-use AppBundle\Repository\UserRepository;
+use App\Entity\GenusScientist;
+use App\Entity\User;
+use App\Repository\UserRepository;
 use Symfony\Bridge\Doctrine\Form\Type\EntityType;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
diff --git a/src/AppBundle/Form/LoginForm.php b/src/Form/LoginForm.php
similarity index 94%
rename from src/AppBundle/Form/LoginForm.php
rename to src/Form/LoginForm.php
index e7f478a..50b05be 100644
--- a/src/AppBundle/Form/LoginForm.php
+++ b/src/Form/LoginForm.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Form;
+namespace App\Form;
 
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\PasswordType;
diff --git a/src/AppBundle/Form/TypeExtension/HelpFormExtension.php b/src/Form/TypeExtension/HelpFormExtension.php
similarity index 94%
rename from src/AppBundle/Form/TypeExtension/HelpFormExtension.php
rename to src/Form/TypeExtension/HelpFormExtension.php
index c4d7291..b85abb0 100644
--- a/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
+++ b/src/Form/TypeExtension/HelpFormExtension.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Form\TypeExtension;
+namespace App\Form\TypeExtension;
 
 use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\Form\Extension\Core\Type\FormType;
diff --git a/src/AppBundle/Form/UserEditForm.php b/src/Form/UserEditForm.php
similarity index 91%
rename from src/AppBundle/Form/UserEditForm.php
rename to src/Form/UserEditForm.php
index 8829b50..ab403d5 100644
--- a/src/AppBundle/Form/UserEditForm.php
+++ b/src/Form/UserEditForm.php
@@ -1,9 +1,9 @@
 <?php
 
-namespace AppBundle\Form;
+namespace App\Form;
 
-use AppBundle\Entity\Genus;
-use AppBundle\Entity\User;
+use App\Entity\Genus;
+use App\Entity\User;
 use Symfony\Bridge\Doctrine\Form\Type\EntityType;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
diff --git a/src/AppBundle/Form/UserRegistrationForm.php b/src/Form/UserRegistrationForm.php
similarity index 94%
rename from src/AppBundle/Form/UserRegistrationForm.php
rename to src/Form/UserRegistrationForm.php
index 1c7577d..ece7177 100644
--- a/src/AppBundle/Form/UserRegistrationForm.php
+++ b/src/Form/UserRegistrationForm.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Form;
+namespace App\Form;
 
-use AppBundle\Entity\User;
+use App\Entity\User;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
 use Symfony\Component\Form\Extension\Core\Type\PasswordType;
diff --git a/src/AppBundle/Repository/GenusNoteRepository.php b/src/Repository/GenusNoteRepository.php
similarity index 87%
rename from src/AppBundle/Repository/GenusNoteRepository.php
rename to src/Repository/GenusNoteRepository.php
index 9bde0e5..c5004a7 100644
--- a/src/AppBundle/Repository/GenusNoteRepository.php
+++ b/src/Repository/GenusNoteRepository.php
@@ -1,9 +1,9 @@
 <?php
 
-namespace AppBundle\Repository;
+namespace App\Repository;
 
-use AppBundle\Entity\Genus;
-use AppBundle\Entity\GenusNote;
+use App\Entity\Genus;
+use App\Entity\GenusNote;
 use Doctrine\ORM\EntityRepository;
 
 class GenusNoteRepository extends EntityRepository
diff --git a/src/AppBundle/Repository/GenusRepository.php b/src/Repository/GenusRepository.php
similarity index 94%
rename from src/AppBundle/Repository/GenusRepository.php
rename to src/Repository/GenusRepository.php
index a77abd8..3fb2d43 100644
--- a/src/AppBundle/Repository/GenusRepository.php
+++ b/src/Repository/GenusRepository.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Repository;
+namespace App\Repository;
 
-use AppBundle\Entity\Genus;
+use App\Entity\Genus;
 use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\EntityRepository;
 
diff --git a/src/AppBundle/Repository/SubFamilyRepository.php b/src/Repository/SubFamilyRepository.php
similarity index 87%
rename from src/AppBundle/Repository/SubFamilyRepository.php
rename to src/Repository/SubFamilyRepository.php
index de614c4..bac4d58 100644
--- a/src/AppBundle/Repository/SubFamilyRepository.php
+++ b/src/Repository/SubFamilyRepository.php
@@ -1,9 +1,9 @@
 <?php
 
-namespace AppBundle\Repository;
+namespace App\Repository;
 
-use AppBundle\Entity\Genus;
-use AppBundle\Entity\SubFamily;
+use App\Entity\Genus;
+use App\Entity\SubFamily;
 use Doctrine\ORM\EntityRepository;
 
 class SubFamilyRepository extends EntityRepository
diff --git a/src/AppBundle/Repository/UserRepository.php b/src/Repository/UserRepository.php
similarity index 90%
rename from src/AppBundle/Repository/UserRepository.php
rename to src/Repository/UserRepository.php
index 3338ef2..2667796 100644
--- a/src/AppBundle/Repository/UserRepository.php
+++ b/src/Repository/UserRepository.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Repository;
+namespace App\Repository;
 
 use Doctrine\ORM\EntityRepository;
 
diff --git a/src/AppBundle/Security/LoginFormAuthenticator.php b/src/Security/LoginFormAuthenticator.php
similarity index 97%
rename from src/AppBundle/Security/LoginFormAuthenticator.php
rename to src/Security/LoginFormAuthenticator.php
index 0a14fa0..912c93e 100644
--- a/src/AppBundle/Security/LoginFormAuthenticator.php
+++ b/src/Security/LoginFormAuthenticator.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Security;
+namespace App\Security;
 
-use AppBundle\Form\LoginForm;
+use App\Form\LoginForm;
 use Doctrine\ORM\EntityManager;
 use Symfony\Component\Form\FormFactoryInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
diff --git a/src/AppBundle/Service/MarkdownTransformer.php b/src/Service/MarkdownTransformer.php
similarity index 96%
rename from src/AppBundle/Service/MarkdownTransformer.php
rename to src/Service/MarkdownTransformer.php
index 0f6f29f..7b74375 100644
--- a/src/AppBundle/Service/MarkdownTransformer.php
+++ b/src/Service/MarkdownTransformer.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Service;
+namespace App\Service;
 
 use Doctrine\Common\Cache\Cache;
 use Knp\Bundle\MarkdownBundle\MarkdownParserInterface;
diff --git a/src/AppBundle/Service/MessageManager.php b/src/Service/MessageManager.php
similarity index 95%
rename from src/AppBundle/Service/MessageManager.php
rename to src/Service/MessageManager.php
index 8dbdbeb..e07e1c2 100644
--- a/src/AppBundle/Service/MessageManager.php
+++ b/src/Service/MessageManager.php
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Service;
+namespace App\Service;
 
 class MessageManager
 {
diff --git a/src/AppBundle/Twig/MarkdownExtension.php b/src/Twig/MarkdownExtension.php
similarity index 90%
rename from src/AppBundle/Twig/MarkdownExtension.php
rename to src/Twig/MarkdownExtension.php
index 95e7eb2..622a4de 100644
--- a/src/AppBundle/Twig/MarkdownExtension.php
+++ b/src/Twig/MarkdownExtension.php
@@ -1,8 +1,8 @@
 <?php
 
-namespace AppBundle\Twig;
+namespace App\Twig;
 
-use AppBundle\Service\MarkdownTransformer;
+use App\Service\MarkdownTransformer;
 
 class MarkdownExtension extends \Twig_Extension
 {
