diff --git a/app/config/services.yml b/app/config/services.yml
index 2ac7f2d..558d48a 100644
--- a/app/config/services.yml
+++ b/app/config/services.yml
@@ -12,3 +12,4 @@ services:
         class: AppBundle\Twig\MarkdownExtension
         tags:
             - { name: twig.extension }
+        arguments: ['@app.markdown_transformer']
diff --git a/src/AppBundle/Twig/MarkdownExtension.php b/src/AppBundle/Twig/MarkdownExtension.php
index df71589..e2b9f07 100644
--- a/src/AppBundle/Twig/MarkdownExtension.php
+++ b/src/AppBundle/Twig/MarkdownExtension.php
@@ -2,8 +2,17 @@
 
 namespace AppBundle\Twig;
 
+use AppBundle\Service\MarkdownTransformer;
+
 class MarkdownExtension extends \Twig_Extension
 {
+    private $markdownTransformer;
+
+    public function __construct(MarkdownTransformer $markdownTransformer)
+    {
+        $this->markdownTransformer = $markdownTransformer;
+    }
+
     public function getFilters()
     {
         return [
@@ -13,7 +22,7 @@ class MarkdownExtension extends \Twig_Extension
 
     public function parseMarkdown($str)
     {
-        return strtoupper($str);
+        return $this->markdownTransformer->parse($str);
     }
 
     public function getName()
