diff --git a/app/Resources/views/admin/genus/list.html.twig b/app/Resources/views/admin/genus/list.html.twig
new file mode 100644
index 0000000..d9b8157
--- /dev/null
+++ b/app/Resources/views/admin/genus/list.html.twig
@@ -0,0 +1,33 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="container">
+        <div class="row">
+
+            <div class="col-xs-12">
+                <div class="pull-right" style="margin-bottom: 10px; margin-top: 20px;">
+                    <a href="{{ path('admin_genus_new') }}" class="btn btn-primary pull-right">Add <span class="fa fa-plus-circle"></span></a>
+                </div>
+
+                <h1>Genuses</h1>
+
+                <table class="table table-striped">
+                    <tr>
+                        <th>Genus</th>
+                        <th>SubFamily</th>
+                        <th>&nbsp;</th>
+                    </tr>
+                    {% for genus in genuses %}
+                        <tr>
+                            <td>{{ genus.name }}</td>
+                            <td>{{ genus.subFamily.name }}</td>
+                            <td>
+                                <a href="#" class="btn btn-xs btn-success"><span class="fa fa-pencil"></span></a>
+                            </td>
+                        </tr>
+                    {% endfor %}
+                </table>
+            </div>
+        </div>
+    </div>
+{% endblock %}
\ No newline at end of file
diff --git a/app/Resources/views/admin/genus/new.html.twig b/app/Resources/views/admin/genus/new.html.twig
new file mode 100644
index 0000000..b7dfb7f
--- /dev/null
+++ b/app/Resources/views/admin/genus/new.html.twig
@@ -0,0 +1,11 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <h1>New Genus</h1>
+            </div>
+        </div>
+    </div>
+{% endblock %}
diff --git a/src/AppBundle/Controller/Admin/GenusAdminController.php b/src/AppBundle/Controller/Admin/GenusAdminController.php
new file mode 100644
index 0000000..f534c75
--- /dev/null
+++ b/src/AppBundle/Controller/Admin/GenusAdminController.php
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+/**
+ * @Route("/admin")
+ */
+class GenusAdminController extends Controller
+{
+    /**
+     * @Route("/genus", name="admin_genus_list")
+     */
+    public function indexAction()
+    {
+        $genuses = $this->getDoctrine()
+            ->getRepository('AppBundle:Genus')
+            ->findAll();
+
+        return $this->render('admin/genus/list.html.twig', array(
+            'genuses' => $genuses
+        ));
+    }
+
+    /**
+     * @Route("/genus/new", name="admin_genus_new")
+     */
+    public function newAction()
+    {
+        // let's go to work!
+    }
+}
\ No newline at end of file
