diff --git a/src/AppBundle/Entity/User.php b/src/AppBundle/Entity/User.php
index 3dc96c9..2ddaa69 100644
--- a/src/AppBundle/Entity/User.php
+++ b/src/AppBundle/Entity/User.php
@@ -7,23 +7,41 @@ use Symfony\Component\Security\Core\User\UserInterface;
 
 class User implements UserInterface
 {
+    private $email;
+
+    // needed by the security system
     public function getUsername()
     {
+        return $this->email;
     }
 
     public function getRoles()
     {
+        return ['ROLE_USER'];
     }
 
     public function getPassword()
     {
+        // leaving blank - I don't need/have a password!
     }
 
     public function getSalt()
     {
+        // leaving blank - I don't need/have a password!
     }
 
     public function eraseCredentials()
     {
+        // leaving blank - I don't need/have a password!
+    }
+
+    public function getEmail()
+    {
+        return $this->email;
+    }
+
+    public function setEmail($email)
+    {
+        $this->email = $email;
     }
 }
