diff --git a/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php b/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
new file mode 100644
index 0000000..f836511
--- /dev/null
+++ b/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
@@ -0,0 +1,23 @@
+<?php
+
+namespace AppBundle\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class AddNiceHeaderEventSubscriber implements EventSubscriberInterface
+{
+    public function onKernelResponse(FilterResponseEvent $event)
+    {
+        $event->getResponse()
+            ->headers->set('X-NICE-MESSAGE', 'That was a great request!');
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            KernelEvents::RESPONSE => 'onKernelResponse'
+        ];
+    }
+}
