diff --git a/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php b/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
index f8365110..7e73d514 100644
--- a/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
+++ b/src/AppBundle/EventSubscriber/AddNiceHeaderEventSubscriber.php
@@ -2,14 +2,24 @@
 
 namespace AppBundle\EventSubscriber;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use Symfony\Component\HttpKernel\KernelEvents;
 
 class AddNiceHeaderEventSubscriber implements EventSubscriberInterface
 {
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     public function onKernelResponse(FilterResponseEvent $event)
     {
+        $this->logger->info('Adding a nice header!');
+
         $event->getResponse()
             ->headers->set('X-NICE-MESSAGE', 'That was a great request!');
     }
