diff --git a/src/AppBundle/Form/TypeExtension/HelpFormExtension.php b/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
new file mode 100644
index 0000000..a0c88a8
--- /dev/null
+++ b/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle\Form\TypeExtension;
+
+use Symfony\Component\Form\AbstractTypeExtension;
+
+class HelpFormExtension extends AbstractTypeExtension
+{
+    public function getExtendedType()
+    {
+        // TODO: Implement getExtendedType() method.
+    }
+}
