diff --git a/app/Resources/views/admin/genus/_form.html.twig b/app/Resources/views/admin/genus/_form.html.twig
index f52ff75..fb80089 100644
--- a/app/Resources/views/admin/genus/_form.html.twig
+++ b/app/Resources/views/admin/genus/_form.html.twig
@@ -18,7 +18,15 @@
         help: 'Should this genus be shown on the site?'
     }) }}
 
-    {{ form_row(genusForm.firstDiscoveredAt.year) }}
+    {{ form_label(genusForm.firstDiscoveredAt.year) }}
+    <select id="{{ genusForm.firstDiscoveredAt.year.vars.id }}"
+            name="{{ genusForm.firstDiscoveredAt.year.vars.full_name }}">
+        {% for choice in genusForm.firstDiscoveredAt.year.vars.choices %}
+            <option value="{{ choice.value }}" {{ choice.value == genusForm.firstDiscoveredAt.year.vars.data }}>{{ choice.label }}</option>
+        {% endfor %}
+    </select>
+    {{ form_errors(genusForm.firstDiscoveredAt.year) }}
+
     {{ form_row(genusForm.firstDiscoveredAt.month) }}
     {{ form_row(genusForm.firstDiscoveredAt.day) }}
 
