diff --git a/src/AppBundle/Form/GenusFormType.php b/src/AppBundle/Form/GenusFormType.php
index 402e0aa..cdb51ce 100644
--- a/src/AppBundle/Form/GenusFormType.php
+++ b/src/AppBundle/Form/GenusFormType.php
@@ -27,7 +27,9 @@ class GenusFormType extends AbstractType
                 }
             ])
             ->add('speciesCount')
-            ->add('funFact')
+            ->add('funFact', null, [
+                'help' => 'For example, Leatherback sea turtles can travel more than 10,000 miles every year!'
+            ])
             ->add('isPublished', ChoiceType::class, [
                 'choices' => [
                     'Yes' => true,
@@ -42,11 +44,6 @@ class GenusFormType extends AbstractType
         ;
     }
 
-    public function finishView(FormView $view, FormInterface $form, array $options)
-    {
-        $view['funFact']->vars['help'] = 'For example, Leatherback sea turtles can travel more than 10,000 miles every year!';
-    }
-
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
diff --git a/src/AppBundle/Form/TypeExtension/HelpFormExtension.php b/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
index 3e5e3c7..c4d7291 100644
--- a/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
+++ b/src/AppBundle/Form/TypeExtension/HelpFormExtension.php
@@ -6,12 +6,20 @@ use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\Form\Extension\Core\Type\FormType;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class HelpFormExtension extends AbstractTypeExtension
 {
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
-        $view->vars['help'] = 'TURTLES!';
+        if ($options['help']) {
+            $view->vars['help'] = $options['help'];
+        }
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefault('help', null);
     }
 
     public function getExtendedType()
